src/reflection.ts

```diff
@@ -428,21 +425,32 @@ export class Parser {
     if (fieldType.baseType() !== reflection.BaseType.Vector) {
       throw new Error("Field " + fieldName + " is not an vector.");
     }
-    if (!isScalar(fieldType.element())) {
+    const elementType = fieldType.element();
+    if (!isScalar(elementType)) {
       throw new Error("Field " + fieldName + " is not an vector of scalars.");
     }
+    const isByteVector =
+      elementType === reflection.BaseType.UByte || elementType === reflection.BaseType.Byte;
```

Comment:
I'm not too familiar with type/javascript conventions around this, but is there a reason not to use Int8Array for the signed Byte?

Reply:
yeah, was trying to cover your comment (vector of bytes), but no real reason other than that. I can scope it to just use uint8array for ubyte.